pkg/storage/chunk/client/ibmcloud/cos_object_client_test.go

```diff
@@ -1,14 +1,54 @@
 package ibmcloud
 
 import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"net/http"
 	"testing"
 
+	"github.com/IBM/ibm-cos-sdk-go/aws/request"
+	"github.com/IBM/ibm-cos-sdk-go/service/s3"
+	"github.com/IBM/ibm-cos-sdk-go/service/s3/s3iface"
+	"github.com/grafana/dskit/backoff"
+
```

Comment:
This line should be removed

Reply:
All these imports are used in the code @adityacs

Reply:
I mean, the empty line `14` should be removed